src/universal/mutations/fragments/UserAnalyticsFrag.js

```diff
@@ -0,0 +1,9 @@
+graphql`
```

Comment:
This is an interesting pattern; declaring fragments in their own modules. Since this fragment is just on the `User` type, I don't see why it should have to live under the `mutations` directory in particular.

Reply:
yeah, but if not here, where?

Reply:
I was just pointing gout that it doesn't need to be under `.../mutations/...`. Idk, `universal/fragments`?